docs/DeveloperGuide.adoc

```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
 == Dev Ops
 
 Refer to the guide <<DevOps#, here>>.
+////
 
 [appendix]
 == Product Scope
+=== Target User Profile
+* Health-conscious people watching their diet
+* Prefer desktop applications over mobile apps
+* Comfortable enough to work with text command-based interfaces
 
-*Target user profile*:
-
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
-
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+=== Value Proposition
+The app is able to help the user keep track of their calorie consumption. This way, they can ensure that they keep to their own diet plans.
 
 [appendix]
 == User Stories
```

Comment:
Can there be a user story that involves storing and displaying the calorie count for regularly had meals before user decides on what to eat?

Reply:
Additionally, it might be helpful to a user if calories burnt were counted too, so that a daily calorie deficit/surplus can be kept track of.